runtime/polkadot/src/xcm_config.rs

```diff
@@ -70,6 +70,8 @@ pub type SovereignAccountOf = (
 	ChildParachainConvertsVia<ParaId, AccountId>,
 	// We can directly alias an `AccountId32` into a local account.
 	AccountId32Aliases<ThisNetwork, AccountId>,
+	// Allow governance body to be used as a sovereign account.
+	HashedDescription<AccountId, DescribeFamily<DescribeBodyTerminal>>,
```

Comment:
Wait sorry, new features like these really should not be deployed immediately on Polkadot -- they should be tested out on Kusama first.

Reply:
This code is tested but I can comment out the changes in Polkadot runtime.

Reply:
How are you going to test this on Kusama? The fellowship setup is different in Kusama. I have e2e tested this already.

Reply:
Test on Kusama, meaning it should be released on Kusama first and have it battle-tested, not simply e2e tested.

Reply:
I know what you mean and how do you do that? The fellowship on Kusama is not on a collective parachain.

Reply:
Then you did not know what I mean:

> any governance body can make use of this converter

Reply:
What you raised is a separate concern from what I mentioned -- new changes should not be immediately deployed on Polkadot. We can discuss how we can test it on Kusama, but that doesn't invalidate the principle of not immediately deploying new changes on Polkadot.